docs/guide/performance.md

```diff
@@ -11,9 +11,8 @@ If you are in a hurry, the most important recommendations on this page are:
 
 - Use PHP 7.x, and dump optimized Composer autoload classmap
 - Use a full web (Nginx/Apache) server with vhost
-- Avoid shared file systems for code _(Docker for Mac/Win, VirtualBox/*, Vagrant, ..)_, or find ways to optimize/workaround the issues.
-- For clustering _(mainly relevant for production/staging)_; reduce latency to Redis/Memcached, use Varnish & [Solr](solr.md)
-
+- Avoid shared filesystems for code (Docker for Mac/Win, VirtualBox/*, Vagrant, etc.), or find ways to optimize or work around the issues.
```

Comment:
note: unsure if readthedoc is affected, but I see everything after ´*´ is italic here

Reply:
Local build looks ok, GitHub also renders it correctly in preview.